app/templates/mandatarissen/persoon/mandataris.hbs

```diff
@@ -105,15 +104,29 @@
   @onStateChanged={{this.onUpdateState}}
   @mandataris={{this.model.mandataris}}
 >
+  <AuAlert
+    @skin="info"
+    @icon="info-circle"
+    @size="small"
+    @closable={{false}}
+    class="au-u-margin-small"
+  >
+    <p><span class="au-u-medium">Wijzig mandataris</span>
+      wordt gebruikt om bijvoorbeeld de fractie te wijzigen of de status van de
+      mandataris aan te passen. Dit zal een nieuw mandaat starten met de
+      gewijzigde situatie.</p>
+  </AuAlert>
   {{#if @model.bestuurseenheid.isOCMW}}
-    <AuAlert @skin="warning" @icon="alert-triangle" @closable={{false}}>
-      Mandatarissen in het OCMW hebben vaak een corresponderend mandaat in de
-      gemeente. Indien aan deze mandataris een corresponderend mandaat gekoppeld
-      is in de gemeente zal je de koppeling breken door hier wijzigingen uit te
-      voeren, in dat geval moet je wijzigingen aan beide mandaten manueel
-      doorvoeren. Zijn ze gekoppeld en voeg je de wijzigingen uit in de
-      gemeente, dan zullen de wijzigingen ook uitgevoerd worden in het OCMW
-      (indien gewenst).
+    <AuAlert
+      @skin="warning"
+      @icon="alert-triangle"
+      @closable={{false}}
+      @size="small"
+      class="au-u-margin-left-small au-u-margin-right-small au-u-margin-bottom-small"
+    >
+      Let op! Gegevens die hier gewijzigd worden zullen niet doorgevoerd worden
+      in het gekoppelde gemeente mandaat. Om een wijziging aan beide mandaten te
+      maken, gelieve dit te doen bij de Gemeente.
```

Comment:
Ik vind dat hier wat nuance mist. Eerst en vooral is er niet per se een gekoppeld mandaat, het kan zijn dat dit er is. Ofwel moeten we die expliciet checken, ofwel moet de warning meer algemeen zijn. Daarbovenop is het niet zo dat enkel de wijziging niet doorgevoerd zal worden in het gekoppelde mandaat, de link zal effectief verbroken worden. En aanpassingen in de gemeente zullen ook niet meer doorstromen naar dit mandaat als het gekoppeld was.